src/app/(proper_react)/redesign/GaScript.test.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect } from "@jest/globals";
+import { render } from "@testing-library/react";
+import { GaScript } from "./GaScript";
+
+afterEach(() => {
+  // Clean up script elements
+  [...document.getElementsByTagName("script")].forEach((element) => {
+    element.remove();
+  });
+});
+
+it("renders the GA script tag", () => {
+  const { container } = render(<GaScript nonce="" />);
+  expect(container).toBeInTheDocument();
+
+  expect(document.getElementsByTagName("script").length).toBe(1);
```

Comment:
I was wondering the same and checked the scripts that were found and logged them while running unit tests: I’ve only caught the GA scripts.

Reply:
To be save in case anything changes in the future or is platform specific I added your suggestion in [2e8821b](https://github.com/mozilla/blurts-server/pull/3402/commits/2e8821b1f7493c5dc4b2c308cb6103ad71cc77e7).